### PR TITLE
Improve profile directory detection by checking if a Player.js file exists.

### PR DESCRIPTION
**Pull request by glneto**
Before loading the profile, I added a verification to see if both `Player.js` and `.profile` files exists.

Also created a test to ensure the function returns null in case the files are not found.

### CODE REVIEW

File: packages/warriorjs-cli/src/Profile.js [hunk 1]
```diff
@@ -1,5 +1,7 @@
 import fs from 'fs';
 import path from 'path';
+import pathType from 'path-type';
```
Comment (olistic) [inline packages/warriorjs-cli/src/Profile.js]: @glneto This import should be in the second group of imports, below `makeDir`, because it's a third-party library.

File: packages/warriorjs-cli/src/Profile.js [hunk 2]
```diff
@@ -41,6 +50,20 @@ class Profile {
     return Object.assign(profile, profileData);
   }
 
+  /**
+   * Verifies if a path is a valid profile directory.
+   *
+   * @param {*} profileDirectoryPath The path to validate.
```
Comment (olistic) [inline packages/warriorjs-cli/src/Profile.js]: `profileDirectoryPath` is of "string" type, right?

File: packages/warriorjs-cli/src/Profile.test.js [hunk 3]
```diff
@@ -38,6 +39,15 @@ describe('Profile.load', () => {
...
+
+  test('returns null if on an invalid profile directory', async () => {
+    Profile.read = () =>
+      'eyJ3YXJyaW9yTmFtZSI6ICJKb2UiLCAidG93ZXJOYW1lIjogImJlZ2lubmVyIiwgImZvbyI6IDQyfQ==';
+    Profile.isProfileDirectory = () => false;
+
+    const profile = await Profile.load('/path/to/profile');
+    expect(profile).toBeNull();
+  });
 });
```
Comment (olistic) [inline packages/warriorjs-cli/src/Profile.test.js]: It would be nice if we had tests for `Profile.isProfileDirectory` that leverage `mock-fs`. We should test the result of that method when both files exist, only one of them exists and none exist.
